app/Services/VCard/ImportVCard.php

```diff
@@ -414,7 +439,12 @@ private function existingContactWithEmail(VCard $entry): ?Contact
                 'contact_field_type_id' => $this->getContactFieldTypeId(ContactFieldType::EMAIL),
             ])->whereIn('data', iterator_to_array($entry->EMAIL))->first();
 
-            if ($contactField) {
+            if ($contactField &&
```

Comment:
We should add a comment above this `if` to give the reader an overview of what this complex `if` does.